fieldservice_person_salary/__manifest__.py

```diff
@@ -17,6 +17,7 @@
         "views/fsm_order_views.xml",
         "views/fsm_order_salary_views.xml",
         "views/fsm_order_salary_menu.xml",
+        "reports/fsm_order_template.xml"
```

Comment:
Cosmetic change, missing comma:
```suggestion
        "reports/fsm_order_template.xml",
```